x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -696,7 +703,12 @@ const loadEndpointsPendingActions = async ({
   }
 
   try {
-    const { data: pendingActions } = await fetchPendingActionsByAgentId(Array.from(agentsIds));
+    const response = await fetchPendingActionsByAgentId(Array.from(agentsIds));
+    if (!response) {
+      return;
```

Comment:
another silent failure? Should we at least throw something so it's caught below?

Reply:
yeah, same question - why is `response` empty? 

maybe you were having issues in tests? if so, it is likely because you did not mock the `KibanaServices` (this module `x-pack/plugins/security_solution/public/common/lib/endpoint_pending_actions/endpoint_pending_actions.ts` which is used by the `fetchPendingActionsByAgentId()` service.

Reply:
Actually, TS also says here that `response` is sometimes `undefined`.

Reply:
My bad, it was the `policyResponse` that was `undefined`. 😅